### DISCUSSION TITLE
Pagging on page admin/users

### DISCUSSION THREAD

**Discussion by mage0se** [dani-garcia/vaultwarden#3973] [Ideas]
Hello,
When we have a lot of users on admin user. The request is so very long ton access a users page.
It's possible to to evolve a api user and  admin page user to make a pagging table and load the user in mode pagging.

Thx you

**Comment by stefan0xC** [comment]
How many users do you have?

**Comment by mage0se** [comment]
I have more than 500 users, and the target is around 1000/1500 users. I have build a infrastructure with 2 front and mysql galera (3 nodes).

Today with 500 users, the page takes a 1 or 2 seconds to display and we feel that it is the full user request that takes a little time. I was thinking that for better fluidity, it might be useful to add paging with a modification to the API that would support pagging. No a full request et pagging in html.

I would have liked to contribute :( but rust is not a language that I master...
